content/experiments/quantum-dice.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Quantum Dice
+author: Russell Huffman
+description: A simple dice roller app using quantum computing to generate random numbers.
+image: https://github.com/JRussellHuffman/quantum-dice/blob/master/assets/header-image.pngs
```

Comment:
This is not really an image but a GitHub page. To serve the actual image use [raw.githack.com](https://raw.githack.com/) to convert from a GitHub URL to the actual file URL. Use the production URL to replace the current one.